libmachine/provision/debian.go

```diff
@@ -44,7 +44,7 @@ func (provisioner *DebianProvisioner) Service(name string, action pkgaction.Serv
 		return err
 	}
 
-	command := fmt.Sprintf("sudo systemctl %s %s", action.String(), name)
+	command := fmt.Sprintf("sudo systemctl %s %s || true", action.String(), name)
```

Comment:
Because systemd was causing issues and exiting at random even though the original issued command exited fine.  @tianon can you shed any more light?